lava-v2-jobs-from-api.py

```diff
@@ -174,6 +174,7 @@ def get_job_params(config, test_config, defconfig, opts, build, plan):
         'context': device_type.context,
         'rootfs_prompt': rootfs.prompt,
         'plan_name': test_plan.name,
+        'job_timeout': config.get('job_timeout')
```

Comment:
Missing trailing comma, no jobs were run on staging because of this.

Can you please, please at least try your changes once locally before submitting a PR?

Reply:
I am sorry I do try my changes in a diff repo then the one to use for commits to keep things clean that's why I have these problems often, I am sorry again.